### PR TITLE
[FIX] *: fix some broken tests

### PR DESCRIPTION
**Pull request by lade-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/tests/test_views.py [hunk 1]
```diff
@@ -4564,6 +4564,14 @@ def test_uncommented_invisible_field(self):
             'stock_intrastat',
             'sale_timesheet_enterprise',
             'helpdesk_repair',
+            'l10n_ch',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/base/tests/test_views.py:4567]: Selection_3858 😄

Comment (lade-odoo) [reply odoo/addons/base/tests/test_views.py:4567]: Yeah I know, the thing is that those are not new addons, they have been there before this test. To my understanding, it's just that some addons were forgotten while creating the whitelist in the first place.

**Comment by lade-odoo** [comment]
@odoo/rd-framework-py Could you review that so that me may have a green build on SH for the OXP ?

**Comment by Xavier-Do** [comment]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo rebase-ff

**Pull request closed without merging**